using-mobile-extensions/adobe-journey-optimizer/README.md

```diff
@@ -219,32 +222,34 @@ func application(_: UIApplication, didRegisterForRemoteNotificationsWithDeviceTo
 
 #### Syntax
 
-```text
+```objc
 public static func setPushIdentifier(_ deviceToken: Data?)
 ```
 
-* _deviceToken_ - A `Data` value denoting the push token.
+| **Variable** | **Type** | **Description** |
+| :----------- | :------- | :-------------- |
+| `deviceToken` | Data | The push token value. |
 
-#### Examples
+#### Example
 
-```text
+```objc
 - (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken { 
     [AEPMobileCore setPushIdentifier:deviceToken];
 }
 ```
 {% endtab %}
 {% endtabs %}
 
-## Next Steps
+## Next steps
 
 * [Track Push Interactions](api-reference.md#addpushtrackingdetails)
 
 ## Configuration keys
 
-To update the SDK configuration programmatically, use the following information to change the Messaging configuration values.
+You can update the SDK configuration, including the Messaging configuration values, programatically by using the following information.
```

Comment:
Should ‘messaging’ be capitalized? Or in a [!UICONTROl] tag?
“You can update the SDK configuration, including the messaging configuration values, programatically by using the following information.”

Reply:
No UINCONTROL tags, since this isn't part of ExL, but I believe Messaging _is_ capitalized, since I see it capitalized everywhere else.